secrets/testing.go

```diff
@@ -0,0 +1,94 @@
+package secrets
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/reddit/baseplate.go/filewatcher"
+)
+
+const (
+	// JWTPubKeyPath is the expected key for the EdgeRequestContext public
+	// key.
+	JWTPubKeyPath = "secret/authentication/public-key"
+
+	testVaultURL   = "vault.reddit.ue1.snooguts.net"
+	testVaultToken = "17213328-36d4-11e7-8459-525400f56d04"
+)
+
+// JWTPubKeySecret is the default EdgeRequestContext public key secret.
+//
+// pubkey copied from:
+// https://github.com/reddit/baseplate.py/blob/db9c1d7cddb1cb242546349e821cad0b0cbd6fce/tests/__init__.py#L12
+var JWTPubKeySecret = GenericSecret{
```

Comment:
Since this is still an exported variable in `secrets` package, the name will likely cause confusion and we should make it more obvious that this is a test pub key, not a real thing. Maybe rename it `TestJWTPubKeySecret` or something like that?